sql/fxa_content_auth_events_v1.sql

```diff
@@ -0,0 +1,70 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-derived-datasets.telemetry.fxa_content_auth_events_v1` AS
+  --
+WITH
+  content as (
+  SELECT
+    jsonPayload.logger,
+    jsonPayload.fields.event_type,
+    jsonPayload.fields.os_name,
+    jsonPayload.fields.os_version,
+    jsonPayload.fields.country,
+    jsonPayload.fields.language,
+    jsonPayload.fields.user_id,
+    jsonPayload.fields.user_properties,
+    jsonPayload.fields.event_properties,
+    timestamp,
+    receiveTimestamp
+  FROM
+    `moz-fx-data-derived-datasets.telemetry.fxa_auth_events_v1`
```

Comment:
Added in explicit project prefix to the table names, which is necessary in view definitions.